softlearning/models/feedforward.py

```diff
@@ -27,7 +27,7 @@ def feedforward_model(input_shapes,
     ]
 
     concatenated = tf.keras.layers.Lambda(
-        lambda x: tf.concat(x, axis=-1)
+        lambda x: tf.concat(x, axis=-2)
```

Comment:
@hartikainen in some cases, we need to concatenate `[None, 1, 2]` shape with `[None, 8, 2]`, so we need some flexibility to account for `axis` argument.